examples/js/loaders/GLTFLoader.js

```diff
@@ -1603,6 +1603,8 @@ THREE.GLTFLoader = ( function () {
 
 				materialParams.transparent = true;
 
+				materialParams.alphaTest = material.alphaCutoff || 0.5;
```

Comment:
We should explicitly add the `0.5` right? Because if the exporter detect 0.5 it could skip the value as it's de default.

```
if ( alphaMode === ALPHA_MODES.MASK ) {

  materialParams.alphaTest = material.alphaCutoff || 0.5;

}
```